Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -199,7 +163,7 @@ protected bool TableExists(ManagedConnection connection, string name)
             }, ReadTransactionMode);
         }
 
-        protected void RunDefaultInitialization(ManagedConnection db)
+        protected void RunDefaultInitialization(SQLiteDatabaseConnection db)
```

Comment:
Can't see where this function is called for your new connections?  Not sure the consequences of missing out these PRAGMAS..